src/types/privatekubectl.ts

```diff
@@ -43,22 +43,9 @@ export class PrivateKubectl extends Kubectl {
       ]
 
       if (addFileFlag) {
-         const filenames = extractFileNames(kubectlCmd)
-
-         const tempDirectory =
-            process.env['runner.tempDirectory'] || os.tmpdir() + '/manifests'
-         eo.cwd = tempDirectory
+         const tempDirectory = getTempDirectory()
+         eo.cwd = path.join(tempDirectory, 'manifests')
          privateClusterArgs.push(...['--file', '.'])
-
-         for (const filename of filenames) {
```

Comment:
It makes more sense to me to keep the moving file to the temp directory in this function